PocketKit/Sources/Sync/PocketSource.swift

```diff
@@ -179,6 +179,19 @@ public class PocketSource: Source {
             completion()
         }
     }
+
+    /// Sends the delete call to Backend, you must still implement the logout and reset functionality.
+    public func deleteAccount() async throws {
+        let result = try await apollo.perform(mutation: DeleteUserMutation())
+
+        guard let errors = result.errors, let firstError = errors.first else {
+            // No error! Yay!
```

Comment:
works great! but I am still able to log in with my deleted account, is this a concern or am I missing something?

Reply:
@cyndichin it takes a min. I just checked and the account I deleted yesterday can not log in.